README.md

```diff
@@ -118,23 +118,22 @@ For additional help, join us in the [Storybook Discord](https://discord.gg/story
-| [viewport](addons/viewport/)                | Change display sizes and layouts for responsive components using Storybook |
-
+| [a11y](https://github.com/storybookjs/storybook/tree/next/addons/a11y/)                        | Test components for user accessibility in Storybook                        |
+| [actions](https://github.com/storybookjs/storybook/tree/next/addons/actions/)                  | Log actions as users interact with components in the Storybook UI          |
+| [backgrounds](https://github.com/storybookjs/storybook/tree/next/addons/backgrounds/)          | Let users choose backgrounds in the Storybook UI                           |
+| [cssresources](https://github.com/storybookjs/storybook/tree/next/addons/cssresources/)        | Dynamically add/remove css resources to the component iframe               |
+| [design assets](https://github.com/storybookjs/storybook/tree/next/addons/design-assets/)      | View images, videos, weblinks alongside your story                         |
+| [docs](https://github.com/storybookjs/storybook/tree/next/addons/docs/)                        | Add high quality documentation to your components                          |
+| [events](https://github.com/storybookjs/storybook/tree/next/addons/events/)                    | Interactively fire events to components that respond to EventEmitter       |
+| [google-analytics](https://github.com/storybookjs/storybook/tree/next/addons/google-analytics) | Reports google analytics on stories                                        |
+| [graphql](https://github.com/storybookjs/storybook/tree/next/addons/graphql/)                  | Query a GraphQL server within Storybook stories                            |
+| [jest](https://github.com/storybookjs/storybook/tree/next/addons/jest/)                        | View the results of components' unit tests in Storybook                    |
+| [knobs](https://github.com/storybookjs/storybook/tree/next/addons/knobs)                      | Interactively edit component prop data in the Storybook UI                 |
+| [links](https://github.com/storybookjs/storybook/tree/next/addons/links/)                      | Create links between stories                                               |
+| [query params](https://github.com/storybookjs/storybook/tree/next/addons/queryparams/)         | Mock query params                                                          |
+| [storyshots](https://github.com/storybookjs/storybook/tree/next/addons/storyshots/)            | Snapshot testing for components in Storybook                               |
+| [storysource](https://github.com/storybookjs/storybook/tree/next/addons/storysource/)          | View the code of your stories within the Storybook UI                      |
+| [viewport](https://github.com/storybookjs/storybook/tree/next/addons/viewport/)                | Change display sizes and layouts for responsive components using Storybook |
```

Comment:
@pmallol checking this it seems that the ones that are leading to 404's are:

 - [design-assets](https://github.com/storybookjs/addon-design-assets)
-  [events](https://github.com/storybookjs/addon-events)
- [cssresources](https://github.com/storybookjs/addon-cssresources)
- [graphql](https://github.com/storybookjs/addon-graphql)
- [queryparams](https://github.com/storybookjs/addon-queryparams)
- [google-analytics](https://github.com/storybookjs/addon-google-analytics)
- [knobs](https://github.com/storybookjs/addon-knobs)

The rest are working as they should. If you could adjust the ones I've mentioned and revert the other ones back as they were we'd appreciate it. It seems that I or @shilman forgot to update the readme when #14831 was addressed and closed.

Reply:
cc @coderkevin